extensions/interactions/ItemSelectionInput/directives/oppia-short-response-item-selection-input.component.spec.ts

```diff
@@ -48,21 +48,12 @@ describe('ShortResponseItemSelectionInput', () => {
     fixture = TestBed.createComponent(ShortResponseItemSelectionInputComponent);
     component = fixture.componentInstance;
     component.answer = '["ca_choices_1"]';
-    component.choices = '[{' +
-      '"_html": "choice 1",' +
-      '"_contentId": "ca_choices_1"' +
-    '}, {' +
-      '"_html": "choice 2",' +
-      '"_contentId": "ca_choices_2"' +
-    '}, {' +
-      '"_html": "choice 3",' +
-      '"_contentId": "ca_choices_3"' +
-    '}]';
+    component.choices = '["choice 1","choice 2","choice 3"]';
```

Comment:
Just wondering ... why are we storing this as a long string? Would it be better to store it as a list of strings instead?

Reply:
I'm not sure if I understand. Are you suggesting we split the string at each individual element using string concatenation (similar to the deleted lines)?

Reply:
This is actually meant to be a string that is further parsed by `JSON.parse()` to obtain a traditional array of strings, which is what we operate on.